pkg/run/run.go

```diff
@@ -262,8 +262,8 @@ func (r *RunOptions) Run(kclient kubernetes.Interface, sclient sonobuoyclient.In
 			ObjectMeta: metav1.ObjectMeta{
 				Name: pkg.CertificationNamespace,
 				Annotations: map[string]string{
-					v1.TolerationsAnnotationKey:  string(tolerations),
-					"openshift.io/node-selector": "node-role.kubernetes.io/tests=",
+					"scheduler.alpha.kubernetes.io/defaultTolerations": string(tolerations),
```

Comment:
@bostrt I may be wrong here but didn't find the "defaultTolerations" object.

Reply:
It took me a while but I did end up finding it yesterday here: https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/plugin/pkg/admission/podtolerationrestriction/admission.go#L56-L60

I'm okay with hardcoding this string into our tool so we don't have to pull in such a large dependency for a single string.